### PR TITLE
Document CloudbackupEnumerateRequest better

### PR DESCRIPTION
**Pull request by pault84**
**What this PR does / why we need it**:
There was some confusion regarding the use of SdkCloudBackupEnumerateRequest
this documents the valid combinations and their expectations.

### CODE REVIEW

File: api/api.proto [hunk 1]
```diff
@@ -1883,15 +1883,27 @@ message SdkCloudBackupDeleteAllRequest {
...
 
-// Defines a request to list the backups stored by a cloud provider
+// Defines a request to list the backups stored by a cloud provider.
+// The following combinations can be used to get cloud backup information:
+//
+// * For a specific volume in current cluster: Set `src_volume_id` to your desired volume id
+// and do not provide `cluster_id` and `all`.
+// * For a specific volume in a specific cluster: Set `src_volume_id` to your desired volume id
+// and specify `cluster_id`.
+// * For a specific volume in all clusteres: Set `src_volume_id` to your desired volume id
```
Comment (lpabon) [inline api/api.proto]: clusteres

File: api/api.proto [hunk 2]
```diff
@@ -1883,15 +1883,27 @@ message SdkCloudBackupDeleteAllRequest {
...
+// and specify set `all` to true, do not provide `cluster_id`.
```
Comment (lpabon) [inline api/api.proto]: 'specify set' should probably be 'set'

File: api/api.proto [hunk 3]
```diff
@@ -1883,15 +1883,27 @@ message SdkCloudBackupDeleteAllRequest {
...
+// * For all volumes in current cluster: do not provide `cluster_id`, `volume_id` and `all`.
```
Comment (lpabon) [inline api/api.proto]: 'and all' or 'or all'?

Comment (pault84) [reply api/api.proto]: i did not want them to think they shouldn't provide volume_id OR 'all' they shouldn't provide any

**Pull request merged**